casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala

```diff
@@ -55,6 +55,12 @@ object NodeLaunch {
 
     def createStoreBroadcastGenesis: F[Unit] =
       for {
+        _ <- if (conf.genesisBlockData.posVaultPubKey.isEmpty)
+              Log[F].warn(
+                "Public key for PoS vault is not set, manual transfer from PoS vault will be disabled (config 'pos-vault-pub-key')"
+              )
+            else
+              ().pure[F]
```

Comment:
It can also be rewritten into 
```scala
Log[F].warn("Public key for PoS vault is not set, manual transfer from PoS vault will be disabled (config 'pos-vault-pub-key')" ).whenA(conf.genesisBlockData.posVaultPubKey.isEmpty)
```